java-spiffe-provider/build.gradle

```diff
@@ -1,5 +1,5 @@
 plugins {
-    id "com.github.johnrengelman.shadow" version "5.2.0"
+    id "com.github.johnrengelman.shadow" version "8.1.1"
```

Comment:
nit: can we declare this dependency version once in the parent build.gradle file and reference it both here and in `java-spiffe-helper/build.gradle`?

Reply:
Good idea.